src/resmom/requests.c

```diff
@@ -1548,6 +1548,9 @@ post_suspend(job *pjob, int err)
 			pjob->ji_qs.ji_svrflags |= JOB_SVFLG_Suspend;
 			(void)job_save(pjob, SAVEJOB_QUICK);
 		}
+		else
+			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO, pjob->ji_qs.ji_jobid,
+				"This job can't be suspended, since the job was in exiting state");
```

Comment:
I think instead of saying "job was in exiting state" print the exact substate the job is in now because of which we are not suspending